src/controllers/application.cr

```diff
@@ -57,9 +57,11 @@ module PlaceOS::Api
       response.headers["X-Request-ID"] = logger.request_id = UUID.random.to_s
     end
 
+    private APPLICATION_JSON = "application/json"
```

Comment:
Is there a need for the constant def? If so (e.g. used extensively elsewhere), might also be worth referencing this in the response.